src/Controller/EnMarche/SearchController.php

```diff
@@ -112,4 +114,58 @@ public function resultsAction(Request $request)
             'errors' => $errors ?? [],
         ]);
     }
+
+    /**
+     * @Route(
+     *     "/tous-les-evenements/{page}",
+     *     requirements={"page"="\d+"},
+     *     defaults={"page"=1},
+     *     name="app_event_all"
+     * )
+     * @Method("GET")
+     */
+    public function allEventsAction(int $page)
+    {
+        $eventRepository = $this->getDoctrine()->getRepository(Event::class);
+
+        $results = $eventRepository->findWithPagination($page > 1 ? SearchParametersFilter::DEFAULT_MAX_RESULTS * $page : 0);
```

Comment:
Can you please use the [parameter](https://github.com/EnMarche/en-marche.fr/blob/master/app/config/config.yml#L15) instead of the constant.